mds/migrations/0038_change_event_record_event_type_options.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.1.11 on 2019-08-16 14:11
+
+from django.db import migrations
+import mds.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("mds", "0037_remove_provider_last_start_time_polled")]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
It's a `post_deploy` migration, so it must happen after the `pre_deploy` migrations. This change is a no-op in postgres, it doesn't care about the list of choices.